java-client/src/test/java/org/opensearch/client/opensearch/integTest/AbstractSearchTemplateRequestIT.java

```diff
@@ -85,6 +85,31 @@ public void testTemplateSearchAggregations() throws Exception {
 
     }
 
+    @Test
+    public void testMultiSearchTemplate() throws Exception {
+        var index = "test-msearch-template";
+        createDocuments(index);
+
+        var searchResponse = javaClient().msearchTemplate(
```

Comment:
If a multi search template example is not present, it would be great to add it to the guides and working samples.